### PR TITLE
Added if lesson in stage one

### PR DESCRIPTION
**Pull request by claudiavmbrito**

### CODE REVIEW

File: tests/stage/one/test_s9_if.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import json
+
+
+async def test_if(test_cli):
+    req_data = json.dumps({'field_1': 'x>18'})
+    res = await test_cli.post('/stage/one/if', data=req_data)
+    assert res.status == 200
+    res_data = await res.json()
+    assert res_data['data']['result'] is True
+    await test_cli.close()
```
Comment (jackey8616) [inline tests/stage/one/test_s9_if.py]: Please read the new [CONTRIBUTING.md](https://github.com/YunOSC/PyFun-Backend/blob/master/CONTRIBUTING.md#provide-a-test) to fit new test code.

File: stage/one/s9_if.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        'Create a if statement that prints that if a person is older than' +
+        '18 years old is old, otherwise the person is young.'
+    ],
+    'code': [
+        'age=20',
+        'if(_____):',
+            'print(\'You are old\')',
+        'else:'
+            'print(\'You are young\')',
+    ],
```
Comment (jackey8616) [inline stage/one/s9_if.py]: The code [here](https://github.com/claudiavmbrito/PyFun-Backend/blob/master/stage/one/s9_if.py#L25,#L31) are wrong.
It sould be like:
```python
'code': [
        'age=20',
        'if(_____):',
        '    print(\'You are old\')',
        'else:'
        '    print(\'You are young\')',
    ],
```
Don't forget the sapce!

File: stage/one/s9_if.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        'tutorial/controlflow.html',
+        '\n',
+        'Now lets work on some exercises. A person is 20 year old age' +
+        'is she young or old?',
+        'Create a if statement that prints that if a person is older than' +
+        '18 years old is old, otherwise the person is young.'
+    ],
+    'code': [
+        'age=20',
+        'if(____):',
```
Comment (jackey8616) [inline stage/one/s9_if.py]: five `_`

Comment (claudiavmbrito) [reply stage/one/s9_if.py]: I think it is six, no?

File: stage/one/s9_if.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        global data, route
+        return route['type'](data, request, answer)
+
+
+def answer(stdout, stderr):
+    try:
+        if stderr != []:
+            return False
+        else:
+            ans = "age>18"
```
Comment (jackey8616) [inline stage/one/s9_if.py]: `You are old` instead `age>18`

**Comment by jackey8616** [comment]
Also, because s9 is already been merged at PR #19 ,
Please re-index your file name to s10.
Thx you!

File: stage/one/s9_if.py (hunk 4 above)
Comment (claudiavmbrito) [reply stage/one/s9_if.py]: oh yes, my bad!

Comment (jackey8616) [reply stage/one/s9_if.py]: Still. `ans = 'You are old\n'`, You forgot line break..

Comment (claudiavmbrito) [reply stage/one/s9_if.py]: sorry, but done

**Comment by jackey8616** [comment]
Good work. 😃 
Thx for your contribution!

**Pull request merged**